source/main.cpp

```diff
@@ -783,12 +475,13 @@ void action_about()
 {
     consoleClear();
 
-    printf(CONSOLE_RED "CIAngel by cearp and Drakia\n" CONSOLE_RESET);
+    printf(CONSOLE_RED "CIAngel by cearp and Drakia\n");
+    printf( CONSOLE_RESET);
```

Comment:
This is an interesting one, actually.. the reason for this change was to stop the app from crashing! :open_mouth: 

Not sure why, but this change actually made a difference.